util/option_functions.R

```diff
@@ -56,6 +56,11 @@ check_options <- function(opt) {
                                               " (", dirname(opt[[option]]), ") ",
                                               "does not exist.")
       }
+    } else if (option == "ncores") {
+      if (!is.integer(opt[[option]]) | opt[[option]] < 1) {
```

Comment:
This seems reasonable! 👍🏻  I was going to comment that the less than 1 check seemed like overkill, but typos happen. I might have a slight preference for taking the minimum in the script itself and if it's less than 1, set `ncores` to 1. I believe what you've done is more consistent with how the options are checked in the codebase, currently.